config/environments/development.rb

```diff
@@ -104,4 +110,13 @@
   config.hosts << ".preview.app.github.dev"
 
   config.active_storage.service = :local
+
+  # Controls whether the PostgresqlAdapter should decode dates automatically with manual queries.
+  #
+  # Example:
+  #   ActiveRecord::ConnectionAdapters::PostgreSQLAdapter.select_value("select '2024-01-01'::date") #=> Date
+  #
+  # This query will return a `String` if postgresql_adapter_decode_dates is set to false.
+  
+  config.active_record.postgresql_adapter_decode_dates = false
```

Comment:
I don't understand what impact this may have 🧐

Reply:
It's in the comments above. I've pasted it in the comments here for convenience.

``` 
# Controls whether the PostgresqlAdapter should decode dates automatically with manual queries.
  #
  # Example:
  #   ActiveRecord::ConnectionAdapters::PostgreSQLAdapter.select_value("select '2024-01-01'::date") #=> Date
  #
  # This query will return a `String` if postgresql_adapter_decode_dates is set to false.
```

It's a new feature in rails 7.2. I've turned it off due to tests failing because of it.

See: https://github.com/DFE-Digital/apply-for-teacher-training/pull/9692/commits/1722b453bb81f2697d6d500b3832ac3913a15f0e

Reply:
Ah I see - the example of the failing test helped. We're asking Postgres to return a date string rather than a Date object 👌 which was the default pre 7.2 behaviour 👌